### PR TITLE
created employee and sales tables

### PR DESCRIPTION
**Pull request by steveh2707**

### CODE REVIEW

File: 001-create-employee-and-sales-employee-tables.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    first_name varchar(70) NOT NULL,
+    last_name varchar(70) NOT NULL,
+    salary decimal(10,2) NOT NULL,
+    bank_account_number varchar(8) NOT NULL,
+    NI_number varchar(9) NOT NULL,
+    PRIMARY KEY (employee_id)
+);
+
+CREATE table sales_employee (
+	sales_employee_id smallint unsigned NOT NULL,
```
Comment (shaunganley) [inline 001-create-employee-and-sales-employee-tables.sql]: is this needed? think about the impact it will have on the data integrity of the table

**Review by shaunganley — CHANGES_REQUESTED** [review]
comment added, can you include some insert and select statements to prove the feature works

File: 001-create-employee-and-sales-employee-tables.sql [hunk 2]
```diff
@@ -0,0 +1,27 @@
+USE ThePrimaryKeys_DylanC;
+
+CREATE table employee (
+	employee_id smallint unsigned NOT NULL AUTO_INCREMENT,
+    first_name varchar(70) NOT NULL,
+    last_name varchar(70) NOT NULL,
+    salary decimal(10,2) NOT NULL,
+    bank_account_number varchar(8) NOT NULL,
+    NI_number varchar(9) NOT NULL,
```
Comment (glenn-hamilton) [inline 001-create-employee-and-sales-employee-tables.sql]: `NI_number` is inconsistent with the rest of rest of the column names

File: 001-create-employee-and-sales-employee-tables.sql [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+);
+
+INSERT INTO employee (first_name, last_name, salary, bank_account_number, NI_number) VALUES ('Stephen', 'Hanna', 50000.00, '23418921', 'JX458205B');
+INSERT INTO employee (first_name, last_name, salary, bank_account_number, NI_number) VALUES ('John', 'Smith', 40000.00, '23123921', 'JX904205B');
+
+INSERT INTO sales_employee (employee_id, commission_rate) VALUES (1, 0.05);
+INSERT INTO sales_employee (employee_id, commission_rate) VALUES (2, 0.05);
+
+SELECT * FROM employee;
+SELECT * FROM sales_employee;
```
Comment (glenn-hamilton) [inline 001-create-employee-and-sales-employee-tables.sql]: Can you not use a wildcard `*` for the select, and merge these into 1 query using a join

**Review by shaunganley — APPROVED** [review]

**Pull request merged**